ThirdParty/Kickstart

```diff
@@ -1 +1 @@
-Subproject commit 5e03d0127e514dcf8a63ae9ebd385cd392dcde52
+Subproject commit 290303cd5bedfdc802582a7cb324de1f0619ab08
```

Comment:
@cra0zy @mrhelmut - Let's take this conversation https://github.com/mono/MonoGame/issues/4895.

@Jjagg - Please remove this change as well.  Looks like we'll be doing something different for it.